### ISSUE TITLE
update dependencies

### ISSUE THREAD

**Issue by dbuentello** [NodejsHouston/sidenote-server#2]
output from `npm outdated`

``` sh
Package       Current  Wanted  Latest  Location
bower          1.3.12  1.3.12   1.7.7  bower
good            5.1.2   5.1.2   6.6.0  good
good-console    4.1.0   4.1.0   5.3.1  good-console
hapi            8.8.1   8.8.1  13.2.1  hapi
```

**Comment by dbuentello** [comment]
if someone wants to tackle this, here is my attempt at it

https://github.com/dbuentello/hapi-plugins.com

**Comment by dbuentello** [comment]
fixed via #5
